src/impl/tpls/KokkosBlas3_gemm_tpl_spec_avail.hpp

```diff
@@ -169,6 +169,60 @@ KOKKOSBLAS3_GEMM_TPL_SPEC_AVAIL_CUBLAS(Kokkos::complex<float>,
                                        Kokkos::LayoutRight,
                                        Kokkos::CudaUVMSpace)
 
+#endif
+
+// rocBLAS
+#ifdef KOKKOSKERNELS_ENABLE_TPL_ROCBLAS
+
+#define KOKKOSBLAS3_GEMM_TPL_SPEC_AVAIL_ROCBLAS(SCALAR, LAYOUTA, LAYOUTB,  \
```

Comment:
I am lazy so I would suggest only having one `LAYOUT` parameter since we always use the same for all the views.
If you foresee that we might need more fined grained control in the future we could just add a second MACRO that calls this one we the same value for `LAYOUTA`, `LAYOUTB` and `LAYOUTC`.

Reply:
Done.